spyder/app/start.py

```diff
@@ -12,10 +12,12 @@
 import sys
 if os.environ.get('PYTHONPATH'):
     for path in os.environ['PYTHONPATH'].split(os.pathsep):
-        try:
-            sys.path.remove(path.rstrip(os.sep))
-        except ValueError:
-            pass
+        if 'pkgs' not in path:
+            # Don't remove pynsist installer entry for 'pkgs' directory
```

Comment:
```suggestion
        if os.name == 'nt' and 'pkgs' in path:
            # Don't remove pynsist installer entry for 'pkgs' directory
            continue
        else:
```

To be more specific about when we avoid removing a directory that contains `pkgs` in its path.